Embed mime-types in executable

matiasgarciaisaia opened the pull request:
camo.cr will default the mime-types list to the one found during compile time if it doesn't found a list at runtime.

This is useful for deploying camo to an external environment without having to copy any dependency at all.

RX14 commented:
How about making it so that it attempts to load a mime-types file specified by ENV var (with compiled-in data fallback), so that the binary is portable outside the system where it was compiled. Probably `CAMO_MIME_FILE` or something similar.

matiasgarciaisaia committed:
Embed mime-types in executable

camo.cr will default the mime-types list to the one found during
compile time, unless a file is specified via CAMO_MIME_TYPES env
var

matiasgarciaisaia commented:
Nice idea 👍

I wasn't sure about including a new ENV var, but it totally makes sense.

I just rewrote it. I don't like calling the `default_accepted_mime_types` in both initializers, but didn't want to load the defaults - and then override them. Not a great penalty to startup time, but anyways.
